Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -13,6 +13,26 @@ final class SubscriptionsViewController: BaseViewController {
 
     @IBOutlet weak var tableView: UITableView!
     @IBOutlet weak var activityViewSearching: UIActivityIndicatorView!
+    @IBOutlet weak var imageViewIcon: UIImageView!
+    @IBOutlet weak var avatarViewContainer: AvatarView! {
+        didSet {
+            if let avatarView = AvatarView.instantiateFromNib() {
+                self.avatarView.frame = avatarViewContainer.bounds
+                avatarViewContainer.addSubview(avatarView)
+                self.avatarView = avatarView
+            }
+        }
+    }
+
+    weak var avatarView: AvatarView! {
+        didSet {
+            avatarView.layer.cornerRadius = 4
+            avatarView.layer.masksToBounds = true
+            avatarView.labelInitialsFontSize = 18
+            avatarView.frame.size.width = 40.0
+            avatarView.frame.size.height = 40.00
```

Comment:
Let's change this to something like this?

```swift
avatarView.frame = CGRect(x: 0, y: 0, width: avatarViewContainer.width, height: avatarViewContainer.height)
```

Reply:
Size adjusted, I changed the size definition for the updateCurrentUserInformation method so I already set the image there